### ISSUE TITLE
`first-published-tag-for-merged-pr` on issues

### ISSUE THREAD

**Issue by fregante** [refined-github/refined-github#5812]
It’s not as straightforward because we’d need to fetch the commit I think, but what do you think about adding the banner on issues that end with “closed by PR”? It’d have to exclude PRs coming from other repos however

**Comment by kidonng** [comment]
Maybe the `cacheKey` of `first-published-tag-for-merged-pr` could become pull request number so we don't need a separate cache for this.

**Comment by yakov116** [comment]
I think this feature is becoming a little too big would need to extract the logic into a new file too.

**Comment by fregante** [comment]
Agreed, I think that a whole new feature can be added just for organization purposes.

`first-published-tag-for-fixed-issue` (so long, but needed for consistency)

Or:

- `tag-after-pr` (rename)
- `tag-after-issue`

The complexity in `tag-after-issue` comes from having to use the API to fetch the merge commit from the "fixed by" PR. In the PRs this piece of information is available on the page:

https://github.com/refined-github/refined-github/blob/a4a7f4839cc95b2e90a26171291b0993a1b25053/source/features/closing-remarks.tsx#L46

**Comment by fregante** [comment]
Implemented by GitHub, e.g. https://github.com/refined-github/refined-github/issues/8473

<img width="922" height="736" alt="Image" src="https://github.com/user-attachments/assets/225920eb-2bf7-4cd4-a56d-290548dc9b18" />

**Comment by SunsetTechuila** [comment]
https://x.com/matthewisabel/status/2042334236070339029

**Comment by fregante** [comment]
We can review after they add it to the PRs, I'm not sure that it's visible enough. We add it to both the header and the footer currently, which feels better than the sidebar
